examples/index.md

```diff
@@ -167,9 +167,11 @@ Content-Type: application/vnd.api+json
 
 Examples of how [error objects](http://jsonapi.org/format/#error-objects) work.
 
-### Responses to failed resource creation:
```

Comment:
I changed this heading because the errors in this section didn't necessarily have to do with creation (they could also be in response to updates). More importantly, though, I tried to structure the examples from simple -> advanced, so starting with a heading that said "Basic..." made sense